molo/profiles/models.py

```diff
@@ -2,6 +2,23 @@
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.contrib.auth.models import User
+from phonenumber_field.modelfields import PhoneNumberField
+from wagtail.contrib.settings.models import BaseSetting, register_setting
+from django.utils.translation import ugettext_lazy as _
+
+
+@register_setting
+class UserProfilesSettings(BaseSetting):
+    show_mobile_number_field = models.BooleanField(
+        default=False,
+        editable=True,
+        verbose_name=_("Add mobile number field to registration"),
+    )
+    mobile_number_required = models.BooleanField(
+        default=False,
+        editable=True,
+        verbose_name=_('Mobile number required'),
+    )
```

Comment:
Can we add a TODO comment here (or somewhere else) for greying out `mobile_number_required` in the UI if `show_mobile_number_field` isn't checked.